### PR TITLE
Chlorine gas canisters

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Added a chlorine gas tank (canister), and added it to the uplink.
Added a canister icon for poisonous gasses.

image

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Comment by atteria** [comment]
The skull may be a little *too* good. It looks very three dimensional, which would be great if it wasn't supposed to be a decal.

**Comment by FluffyGhoster** [comment]
> The skull may be a little _too_ good. It looks very three dimensional, which would be great if it wasn't supposed to be a decal.

I have just reduced the skull of the image for poisonous gasses:
image
If you can produce a better icon for the tank, I am happy to include it

File: code/datums/uplink/bioweapons.dm [hunk 1]
```diff
@@ -45,3 +45,11 @@
 	telecrystal_cost = 3
 	bluecrystal_cost = 3
 	path = /obj/item/storage/box/syndie_kit/syringe_gun
+
+/datum/uplink_item/item/bioweapons/chlorine_tank
+	name = "Chlorine Gas Tank"
+	desc = "A Chlorine gas tank, a poisonous gas firstly mass produced on Earth for warfare uses, thanks to the research of Haber. At a premium, you can now wage \
+	your personal chemical warfare. Make sure the wind blows in the right direction, and wear the appropriate PPEs."
+	telecrystal_cost = 5
+	bluecrystal_cost = 5
```
Comment (NonQueueingMatt) [inline code/datums/uplink/bioweapons.dm:54]:
```suggestion
```
Not exactlyy a support item.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**